### PR TITLE
preparation of JOSS submission

### PR DESCRIPTION
**Pull request by mommermi**
This PR adds information required by the Journal of Open Source Software for our submission to them, which will enable us to get a cross-ref for this project that people are able to cite. This will be part of our v0.1 release which we aim for by the end of March 2019. 

Please review and comment the information as you wish. This discussion is open to everyone. 

For details on the JOSS submission and review process, please look at:
https://joss.readthedocs.io/en/latest/submitting.html#submitting-a-paper-to-joss

My goal is to submit this to JOSS by the end of March 2019. After that, this PR will be merged and discussions will close. 

**Edit**: for a nicely formatted version of the abstract, please have a look at: https://github.com/mommermi/sbpy/blob/joss/publications/joss/paper.md

### CODE REVIEW

**Comment by mkelley** [comment]
Gave it a first read :+1:

File: publications/joss/paper.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: 'sbpy: A Python module for small-body planetary astronomy'
+tags:
+  - python
+  - astronomy
+  - solar system
+  - planetary science
+  - asteroids
+  - comets
```
Comment (jianyangli) [inline publications/joss/paper.md]: This is probably nitpicking, but doesn't small bodies also include meteoroids, centaurs, and KBOs?  Should we tag `sbpy` to include all small body categories?

**Comment by jianyangli** [comment]
Reads very well to me, too!

File: publications/joss/paper.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+observations, as well as the analysis and interpretation of the
+results, require tailored techniques and models that are disparate
+from those used most other astronomical fields of research. Currently,
+there is no single open source software package available to support
+small-body planetary astronomers in their study of asteroids and
+comets in the same way in which ``Astropy`` [@astropy; @astropy2]
+supports the general astronomy community.
+
+``sbpy`` is a community effort to build a Python package for
+small-body planetary astronomy in the form of an ``Astropy` affiliated
```
Comment (migueldvb) [inline publications/joss/paper.md]:
```
an ``Astropy` affiliated
```
should have double closing backtick characters

File: publications/joss/paper.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+that of ``Astropy`` wherever possible in order to enable user-friendly
+and consistent access to its functionality.
+
+``sbpy`` is setup in a highly modular class-based fashion, roughly
+separated into the individual tasks listed above. The following
+schematic provides an overview of these modules and their
+interrelations, as indicated through colored symbols of different
+shapes:
+
+![``sbpy`` module structure.](structure.png)
```
Comment (migueldvb) [inline publications/joss/paper.md]: Looks great! I could not get the image with the compilation command https://github.com/openjournals/whedon/blob/25f9a1307a83b6b89080d6d934a3621f6a244035/lib/whedon/processor.rb#L101-L122 , but I suspect that something has changed in the building process and that command may not be up to date, so better to leave this issue to the editors.

File: publications/joss/paper.md (hunk 2 above)
Comment (mommermi) [reply publications/joss/paper.md]: Got it, thanks!

File: publications/joss/paper.md (hunk 3 above)
Comment (mommermi) [reply publications/joss/paper.md]: I agree!

**Comment by mommermi** [comment]
Thanks, guys! I implemented all your comments so far. I will leave this open until Mar 31 and submit then if there are no objections.

File: publications/joss/paper.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+constants, units, and modeling submodules, and other affiliated
+packages like [@astroquery]. ``sbpy``'s API is deliberately emulating
+that of ``Astropy`` wherever possible in order to enable user-friendly
+and consistent access to its functionality.
+
+``sbpy`` is setup in a highly modular class-based fashion, roughly
+separated by the individual tasks listed above. The following
+schematic provides an overview of these modules and their
+interrelations as indicated through colored symbols. The main modules,
+shown as black boxes, interact through data countainers for
```
Comment (migueldvb) [inline publications/joss/paper.md]: countainers -> containers

Comment (mommermi) [reply publications/joss/paper.md]: ha, good catch!

**Pull request merged**